cloud/store/schema.go

```diff
@@ -188,11 +191,31 @@ func (s *schema) updateTenants(class string, req *command.UpdateTenantsRequest)
 	return
 }
 
+func (s *schema) clear() {
+	for k := range s.Classes {
```

Comment:
do you think we would need lock here , to be thread safe ? i am not sure when clear will be called, so far i see it exists for tests only, if that it the case we can move it to the test file instead 🤔

Reply:
Done